routers/repo/pull_review.go

```diff
@@ -63,6 +63,10 @@ func CreateCodeComment(ctx *context.Context, form auth.CodeCommentForm) {
 			}
 		}
 	}
+	reviewID := review.ID
```

Comment:
Why are you putting this in an extra variable instead of directly modifiying `review.ID`?